test/metabase/api/user_test.clj

```diff
@@ -303,7 +303,7 @@
                    (dissoc :is_qbnewb :last_login))
                (-> (mt/user-http-request :rasta :get 200 "user/current")
                    mt/boolean-ids-and-timestamps
-                   (dissoc :is_qbnewb :last_login :has_question_and_dashboard))))))
+                   (dissoc :is_qbnewb :has_question_and_dashboard :last_login))))))
     (testing "check that `has_question_and_dashboard` is `true`."
```

Comment:
I think this test should still test the new behavior

Reply:
Do you know how it was working before? I thought that the issue was that it always returns `false`

Reply:
Ah rip (db/exists? 'Card nil) returns true but (t2/exists? 'Card nil) returns false